.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,54 @@
+name: GitHub Pages
```

Comment:
This does have a bit of duplication with the main CI workflow. Ideally we could have a workflow with two jobs: a `build` job and a `deploy` job, which share the results of building the source and the second just bundles and publishes. On the other hand, I know that it's super easy to get bogged down in CI so I don't want to force you to do that. So I'm going to accept this as-is, but if you would like to try merging them together then I welcome that as a further addition.